iosApp/iosApp/Localizable.xcstrings

```diff
@@ -1016,6 +1016,10 @@
         }
       }
     },
+    "Arriving" : {
+      "comment" : "Status for a commuter rail train",
+      "extractionState" : "manual"
+    },
```

Comment:
We could probably plan to rewrite overridden-to-"Arriving" as the existing`ARR` status - and we could probably turn "Now boarding" into `BRD`, too.

Reply:
Yeah, I was thinking that would probably be the way to go, but might as well request them in case.